src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
```

Comment:
Hi Vishnu, a small nit but you might want to consider adding some header Javadocs for this class